cli/admin/src/test/java/io/pravega/cli/admin/dataRecovery/DataRecoveryTest.java

```diff
@@ -1714,6 +1718,29 @@ public void testLTSRecoveryCommand() throws Exception {
         Assert.assertNotNull(RecoverFromStorageCommand.descriptor());
     }
 
+    @Test
```

Comment:
do you want to define a `timeout` for this?

Reply:
everything is locally defiend now in the test, so is quick.